### ISSUE TITLE
.../views/vendor/backpack/logmanager" directory does not exist.

### ISSUE THREAD

**Issue by jackfruh** [Laravel-Backpack/LogManager#27]
# Bug report

### What I did:
`php artisan view:clear`
`php artisan view:cache`

### What I expected to happen:
refresh cache

### What happened:
Error: The "/Volumes/CaseSensitive/lucas57/resources/views/vendor/backpack/logmanager" directory does not exist.

### What I've already tried to fix it:
created empty directory ../views/vendor/backpack/logmanager

### Backpack, Laravel, PHP, DB version:
latest

### Root cause analysis:
I believe github (and possibly git) will not allow an empty folder to be checked into a repository.
It seems very likely that the developer's original test environment would have this empty directory.

An easy fix would be to create the directory and put a readme in it.
I will attempt to do a pull request for this.

**Comment by jackfruh** [comment]
Ok unfortunately I don't understand enough about package management / composer to know what folder in this repo would ultimately translate to `projectroot/resources/views/vendor/backpack/logmanager`

I tried looking at the crud repo since I have a crud folder but had no luck there.

So this may be an easy fix or it may require a code change to check for the existence of the directory first (which is hopefully also easy)

**Comment by tabacitu** [comment]
Hi @jackfruh ,

You're totally right, thank you for reporting this and trying to fix it - we were totally unaware this package broke ```php artisan view:cache```.

I believe the Laravel command itself tested that the folder exists in the past, but it no longer does so in the current version. So I've just pushed a hot fix, we check for this ourselves and no longer use that folder for views, if it doesn't exist. This should be a non-breaking fix, so a ```composer update``` should fix it for you.

I also discovered the same issue in ```BackupManager``` will push an update for that one shortly.

Thanks a lot, cheers!
